docs/contributing/index.md

```diff
@@ -57,8 +57,7 @@ project. Please read this section carefully if you are interested in contributin
 	*   Please make sure your contributions follow our Style Guide.  At a minimum, ensure your source passes through
 	`npm run lint` with no warnings.
 	*   Please squash pull requests down to a single commit to simplify review and keep history clean.
-	*   Ensure all new features have unit tests that pass and any bug fixes include tests that
-	fail without the fix applied
+	*   Ensure all new features have unit tests,[UI-Test](./ui-testing.md) and [Screenshot](./screenshot-testing.md) that pass and any bug fixes include tests that fail without the fix applied.
```

Comment:
```suggestion
	*   Ensure all new features have unit tests,[ui test](./ui-testing.md) and [screenshot test](./screenshot-testing.md) that pass and any bug fixes include tests that fail without the fix applied.
```

Reply:
@jeonghee27 Do you have any specific reason for using terms "UI-Test" and "Screenshot"?